src/_langs/en/updates/2015-08-10-easily-duplicate-dom-nodes.markdown

```diff
@@ -0,0 +1,44 @@
+---
+rss: false
+layout: update
+published: true
+title: Easily duplicate DOM nodes
+date: 2015-08-10
+article:
+  written_on: 2015-08-10
+  updated_on: 2015-08-10
+authors:
+- umarhansa
+collection: updates
+type: tip
+category: tools
+product: chrome-devtools
+featured-image: /web/updates/images/2015-08-10-duplicate-dom-nodes/gplus-8f4a2ba4.png
+source_name: DevTips
+source_url: https://umaar.com/dev-tips/58-duplicate-dom/
+teaserblocks:
+- heading: ""
+  description: ""
+  image: ""
+- heading: ""
+  description: ""
+  image: ""
+- heading: ""
+  description: ""
+  image: ""
+permalink: /updates/2015/08/10/easily-duplicate-dom-nodes.html
+---
+<img src="/web/updates/images/2015-08-10-duplicate-dom-nodes/duplicate-dom.gif" alt="Duplicate DOM nodes">
+
+<em>Or what are essentially cut/copy &amp; paste operations.</em>
+
+You can easily change the DOM without having to edit the HTML as a big piece of string.
+
+<ul>
+<li>Right click on a node and select <em>Copy</em>.</li>
+<li>You can paste in your code editor, or for prototyping, you can paste the DOM node elsewhere in the DOM tree. The pasted node is inserted as a child of the currently selected node. In the video, I use the left arrow key to jump to the immediate parent opening tag and paste there (which is what I do in most cases).</li>
+</ul>

```

Comment:
You don't need to fix this, but ordered lists are done like this in markdown:

1. This is a step.
1. This is the next step.